### PR TITLE
Update Animals Table

### PR DESCRIPTION
**Pull request by lily4178993**
# Update Animals Table

## Descritpion

The purpose of this pull request is to enhance the Vet Clinic Database project by introducing new features and improving the overall structure.

### Changes Made

This pull request includes the following changes:
- Added a new 'species' column to the 'animals' table.
- Inserted additional animal data into the 'data.sql' file.
- Implemented SQL transactions for testing and data updates.
- Added SQL queries to answer specific questions about the 'animals' data.

### Screenshots

Capture d’écran (423)
Capture d’écran (424)
Capture d’écran (425)
Capture d’écran (426)
Capture d’écran (427)
Capture d’écran (428)
Capture d’écran (429)

### Reviewer Note

Please review the changes and provide feedback on the added features, data updates, and query implementations.

Thank you for your time and review!

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -31,3 +31,122 @@ WHERE name != 'Gabumon';
...
+SELECT COUNT(*) AS never_tried_to_escape FROM animals
+WHERE escape_attempts = 0;
+
+-- Find the average weight of animals
+SELECT AVG(weight_kg) AS average_weight FROM animals;
+
+-- Animal who escapes the most, neutered or not neutered animals
+SELECT name, MAX(escape_attempts) AS max_escape_attempts
+FROM animals
+GROUP BY name;
```
Comment (usorfaitheloho) [inline queries.sql]: You did well with this project, kindly make few changes with this query as stated below, the query is to fid out either `neutered` or ` not neutered` :

`  SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
`

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @lily4178993  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights

> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (lily4178993) [reply queries.sql]: Noted @usorfaitheloho.
Thank you for you review and feedback.

**Comment by lily4178993** [comment]
I've updated the code for the question _Who escapes the most, neutered or not neutered animals?_

**Screenshot of the result**
Capture d’écran (430)

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @lily4178993 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors 🟢 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you @codepantha for your review and feedback.

**Pull request merged**